homeassistant/components/nut/const.py

```diff
@@ -114,18 +115,21 @@
         name="Load Start Timer",
         native_unit_of_measurement=TIME_SECONDS,
         icon="mdi:timer-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
     ),
     "ups.timer.reboot": SensorEntityDescription(
         key="ups.timer.reboot",
         name="Load Reboot Timer",
         native_unit_of_measurement=TIME_SECONDS,
         icon="mdi:timer-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
```

Comment:
May be I'm wrong, but in [documentation](https://developers.home-assistant.io/docs/core/entity#generic-properties) `config` is for an entity which allows changing the configuration of a device, for example a switch entity. So why assign config to a read-only sensor? It is because its value can be changed in UPS? And is this the meaning for category?

Reply:
I based the changed based on this https://networkupstools.org/docs/developer-guide.chunked/apas01.html
The integration of NUT doesn't give you the option to change them no, but these are theoretically configurable in the nut config file. I also see there is a` system` category I wasn't aware of. I'm now wondering maybe all these that I put as `config` entities should be `system` instead?

Reply:
`system` is something that user should not consider, a sensor used by the system but not useful information, so I don't think make sense here. Probably all this should be `diagnostic`, but I understand your consideration. Let see what think about this member team.

Reply:
Reading explanation in [beta release note](https://rc.home-assistant.io/blog/2021/10/27/release-202111/)  seems more logic set status sensors without category and all others (or most of them) with category `diagnostic`. Take a look and let me know.

Reply:
Hm yeah perhaps you're right. The problem would then be how to choose which ones are `config` and which ones are `diagnostic`.

Reply:
Imho:

No category:
- Status
- Status Data
- Battery Charge
- Load
- Input Voltage
- Output Voltage

System:
- UPS Type

Diagnostics:
- All others sensors

Reply:
I don't see anyone else suggesting another idea so I'll just change them to the above

Reply:
Would we then have no entities as `config`? Originally there were three, but based on your argument those should also be `diagnostic`

Reply:
I think that `config` should be only for secondary entity that allow configuration. For example, if in the future we will create a switch do disable alarm, this would be `config`.